cost/google/unutilized_ip_addresses/google_unutilized_ip_addresses.pt

```diff
@@ -136,7 +145,7 @@ script "js_get_addresses", type: "javascript" do
       "auth": "auth_google",
       "host": "www.googleapis.com",
       "verb": "GET",
-      "path": "/compute/v1/projects/"+project+"/regions/"+region+"/addresses",
+      "path": "/compute/beta/projects/"+project+"/regions/"+region+"/addresses",
```

Comment:
is this the only endpoint?  should we be using a beta endpoint

Reply:
This is the only endpoint if we want to use labels to filter...